com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -87,8 +87,13 @@ public ForwardRenderer(ForwardRendererData data) : base(data)
             m_CopyDepthPass = new CopyDepthPass(RenderPassEvent.AfterRenderingSkybox, m_CopyDepthMaterial);
             m_DrawSkyboxPass = new DrawSkyboxPass(RenderPassEvent.BeforeRenderingSkybox);
             m_CopyColorPass = new CopyColorPass(RenderPassEvent.AfterRenderingSkybox, m_SamplingMaterial, m_BlitMaterial);
-            m_TransparentSettingsPass = new TransparentSettingsPass(RenderPassEvent.BeforeRenderingTransparents, data.shadowTransparentReceive);
-            m_RenderTransparentForwardPass = new DrawObjectsPass("Render Transparents", false, RenderPassEvent.BeforeRenderingTransparents, RenderQueueRange.transparent, data.transparentLayerMask, m_DefaultStencilState, stencilData.stencilReference);
+#if ADAPTIVE_PERFORMANCE_2_0_0_OR_NEWER
+            if (!UniversalRenderPipeline.asset.useAdaptivePerformance || AdaptivePerformance.AdaptivePerformanceRenderSettings.SkipTransparentObjects == false)
```

Comment:
Does the render pipeline constructor gets called if adaptive performance setting changes?
Otherwise you might get into situation that you get a null reference if the constructor was called with skip transparents and runtime transparents were enabled but the constructor was not called. 

Otherwise if you want to simplify the transparent passes could always be created, they don't consume noticeable memory.